### DISCUSSION TITLE
一个wiki多次添加

### DISCUSSION THREAD

**Discussion by dongrentianyu** [tiddly-gittly/TidGi-Desktop#584] [Ideas]
![image](https://github.com/user-attachments/assets/5afc8ee2-6b48-4751-b629-7209fad58d0d)

刚排查问题过程中，发现太记可以把一个wiki多次添加。正常不应该一个文件夹路径一个工作区吗？有些好奇。

**Comment by linonetwo** [comment]
这样可以让其中一个只读，然后用 cpolar 内网穿透分享成只读博客。然后可读写的那个正常录入信息。

**Reply by dongrentianyu** [reply]
原来是这样，那太记也太好了吧。
